### DISCUSSION TITLE
"This song cannot be played due to server restrictions"

### DISCUSSION THREAD

**Discussion by declan-watson** [fast4x/RiMusic#376] [Q&A]
Hi! This doesn't seem like a rimusic bug (seems more server/YouTube issue) so didn't want to raise an issue but was very curious if anyone else was seeing this error for the song below or any others?

It only really seems to appear when I try to play Killing in the Name by Rage Against the Machine, although I haven't tested a lot of songs it's the only one that consistently doesn't work.

I was thinking it might be something to do with it being explicit and needing age verification but other explicit songs play just fine.

RiMusic v0.6.16.1 on Android 14 Pixel 7 pro

I've tried clearing all cache and storage and hasn't fixed the issue.

**Comment by fast4x** [comment]
Hi, can share link of Song?

**Reply by declan-watson** [reply]
Sure! Thanks for the response 

https://music.youtube.com/watch?v=2o9aoL0NWpw

**Reply by fast4x** [reply]
The song on YouTube is classified with the message "We recommend viewing to an adult audience". Requires confirmation of user acceptance to be reproduced. 
Currently this condition is not managed by RiMusic.

**Comment by fast4x** [comment]
Added to wiki faqs
